src/droplet.F

```diff
@@ -2482,7 +2493,7 @@ subroutine droplet_injection(pdata,xf,yf,dt,s10_xym,randomNumbers)
 
       my_reintro = floor(xrange*yrange*dt*totdrops/drop_mult)      !Number of droplets to be produced by this MPI task
 
-      my_reintro = 10
+      !JMD my_reintro = 10
```

Comment:
remove this line (also removed in David's PR)

Reply:
Done